public/app/features/datasources/NewDataSourcePage.tsx

```diff
@@ -151,17 +142,18 @@ const DataSourceTypeCard: FC<DataSourceTypeCardProps> = props => {
       </div>
       <div className="add-data-source-item-actions">
         {learnMoreLink && (
-          <a
-            className="btn btn-inverse"
+          <LinkButton
+            variant="secondary"
             href={`${learnMoreLink.url}?utm_source=grafana_add_ds`}
             target="_blank"
             rel="noopener"
             onClick={onLearnMoreClick}
+            icon={<Icon name="external-link" />}
```

Comment:
Any thoughts about this usage?

Reply:
can't we just set icon to "external-link"

Reply:
It originally supported setting it as `fa fa-external-link` I think. Should maybe remove the `fa` then, and do some reasonable migration.

Reply:
I'll use the fa string for now, and we'll see about that later.